### PR TITLE
[SYCL] Update docs/comments post llvm-test-suite move

### PR DESCRIPTION
**Pull request by aelovikov-intel**

### CODE REVIEW

**Review by v-klochkov — APPROVED** [review]

File: sycl/doc/GetStartedGuide.md [hunk 1]
```diff
@@ -533,10 +533,10 @@ architecture. It is possible to change it by adding
 `-Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch=<target>` to the CMake
 variable `SYCL_CLANG_EXTRA_FLAGS`.
 
-#### Run DPC++ E2E test suite
+#### Run DPC++ E2E tests
 
 Follow instructions from the link below to build and run tests:
-[README](https://github.com/intel/llvm-test-suite/tree/intel/SYCL#execution)
+[README](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/README.md)
```
Comment (bader) [inline sycl/doc/GetStartedGuide.md:539]:
```suggestion
[README](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/README.md#build-and-run-tests)
```

File: sycl/doc/developer/ContributeToDPCPP.md [hunk 2]
```diff
@@ -144,12 +144,13 @@ When adding new test to `check-sycl`, please consider the following:
 
 ### DPC++ end-to-end (E2E) tests
 
-These tests are extension to
-[LLVM test suite](https://github.com/intel/llvm-test-suite/tree/intel/SYCL)
-and hosted at separate repository.
+These tests are located in [test-e2e](/sycl/test-e2e) directory and are not
```
Comment (bader) [inline sycl/doc/developer/ContributeToDPCPP.md:147]:
```suggestion
These tests are located in [/sycl/test-e2e](/sycl/test-e2e) directory and are not
```

File: sycl/doc/developer/ContributeToDPCPP.md [hunk 3]
```diff
@@ -144,12 +144,13 @@ When adding new test to `check-sycl`, please consider the following:
...
+configured to be run by default. See
+[End-to-End tests documentation](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/README.md)
```
Comment (bader) [inline sycl/doc/developer/ContributeToDPCPP.md:149]:
```suggestion
[End-to-End tests documentation](/sycl/test-e2e/README.md)
```

File: sycl/doc/extensions/experimental/sycl_ext_intel_esimd/esimd_emulator.md [hunk 4]
```diff
@@ -92,39 +92,35 @@ so they can't be used simultaneously by a SYCL offload application process. On t
...
  ..
 
 # Build and Run
-ninja check
+ninja check-sycl-e2e
 
 ```
 
-Note that only [ESIMD Kernels](https://github.com/intel/llvm-test-suite/tree/intel/SYCL/ESIMD) are
+Note that only [ESIMD Kernels](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/ESIMD/) are
```
Comment (bader) [inline sycl/doc/extensions/experimental/sycl_ext_intel_esimd/esimd_emulator.md:123]:
```suggestion
Note that only [ESIMD Kernels](/sycl/test-e2e/ESIMD/) are
```

File: sycl/doc/extensions/experimental/sycl_ext_intel_esimd/esimd_emulator.md [hunk 5]
```diff
@@ -73,7 +73,7 @@ List of target platforms supported by CM_EMU is as follows
 Compilation step for ESIMD kernels prepared for ESIMD_EMULATOR backend
 is same as for OpenCL and Level Zero backends. Full runnable code
 sample used below can be found on the [github
-repo](https://github.com/intel/llvm-test-suite/blob/intel/SYCL/ESIMD/vadd_usm.cpp).
+repo](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/ESIMD/vadd_usm.cpp).
```
Comment (bader) [inline sycl/doc/extensions/experimental/sycl_ext_intel_esimd/esimd_emulator.md:76]:
```suggestion
repo](/sycl/test-e2e/ESIMD/vadd_usm.cpp).
```

File: sycl/doc/extensions/experimental/sycl_ext_intel_esimd/esimd_emulator.md [hunk 6]
```diff
@@ -92,39 +92,35 @@ so they can't be used simultaneously by a SYCL offload application process. On t
 hand, it is OK to mix the emulator with non-Intel GPU devices or CPU device in
 `ONEAPI_DEVICE_SELECTOR`.
 
-## Running ESIMD examples from [ESIMD test suite](https://github.com/intel/llvm-test-suite/tree/intel/SYCL/ESIMD) on github with ESIMD_EMULATOR backend
+## Running ESIMD examples from [ESIMD test suite](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/ESIMD/) on github with ESIMD_EMULATOR backend
```
Comment (bader) [inline sycl/doc/extensions/experimental/sycl_ext_intel_esimd/esimd_emulator.md:95]:
```suggestion
## Running ESIMD examples from [ESIMD test suite](/sycl/test-e2e/ESIMD/) on github with ESIMD_EMULATOR backend
```

**Review by bader — APPROVED** [review]

**Review by AlexeySachkov — APPROVED** [review]

**Comment by aelovikov-intel** [comment]
@intel/llvm-gatekeepers  , this PR is ready.

**Pull request merged**